src/Fixer/PhpUnit/PhpUnitStaticAssertionCallsFixer.php

```diff
@@ -0,0 +1,300 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\PhpUnit;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Indicator\PhpUnitTestCaseIndicator;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class PhpUnitStaticAssertionCallsFixer extends AbstractFixer
+{
+    private static $staticMethods = [
```

Comment:
big :-1: for having this list.
it's easy to use, yet it will be a nightmare to maintain, with every new phpunit release, whenever new method is exposed, one will need to add it to this list...

Reply:
I agree, but what are the solution? PhpUnit may not be in the autoloaded dependencies.

Reply:
maybe there is no solution before adding context-aware fixers

Reply:
> will be a nightmare to maintain

> PhpUnit may not be in the autoloaded dependencies.

What about writing test if all of assertions from PHPUnit installed during Travis job are on the list. It will result with surprising test fail, but with good message will be easy to fix. Not the cleanest solution, but will do the job.

Reply:
It would fail on a version and pass on another, or viceversa

Reply:
No, it won't, it would check if all of assertions from version are on the list and raise and error if any is missing, not when there is any extra one.

Reply:
If a method is in this fixer but not in phpunit, I would like to not fix it. In other words, I want to stay updated not only for new methods, but also for deprecated and removed ones.
Still, we can start with the test you propose, better than nothing.